CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## next
 
+* First pass at describing (and doc-generating) API global information
+  * Inside a `Praxis::ApiDefinition.define` block one can now specify a few things about the API by using:
+    * info("1.0") `<block>` - Which will apply to a particular version only
+    * info `<block>` - Which will be inherited by any existing API version
+    * The current pieces of information that can be defined in the block are: `name`, `title`, `description` and `basepath`. See [this](https://github.com/rightscale/praxis/blob/master/spec/spec_app/design/api.rb) for details
+  * NOTE: This information is output to the JSON files, BUT not used in the doc browser yet.
+* Changed the doc generation and browser to use "ids" instead of "names" for routes and generated files.
+  * Currently, "ids" are generated using dashes instead of double colons (instead of random ids). This closes issue #31.
+* Added the definition and handling of canonical urls for API resources
+  * One can now specify which action URL to be considered as the canonical resource href:
+    * by using `canonical_path_action <action_name>` at the top of the resource definition class
+    * or by using `canonical_path` within the action definition itself.
```

Comment:
Additionally, if we only supported specifying the canonical action at the resource level, we could remove the check that guards against defining it twice (which I assume is there mostly to guard against two actions competing to define it), and support inheriting the action choice from a base `Concern`.